com.unity.netcode.gameobjects/Editor/CodeGen/INetworkMessageILPP.cs

```diff
@@ -99,10 +99,13 @@ private void ProcessINetworkMessage(TypeDefinition typeDefinition)
                     typeSequence = methodSequence;
                 }
                 if (resolved.IsStatic && resolved.IsPublic && resolved.Name == "Receive" && resolved.Parameters.Count == 2
-                    && resolved.Parameters[0].ParameterType.IsByReference
-                    && resolved.Parameters[0].ParameterType.GetElementType().Resolve() ==
+                    && !resolved.Parameters[0].IsIn
```

Comment:
This is the change to the Receive() parameters: `ref` removed from the first parameter, `in` added to the second.